common/src/main/proto/pb.proto

```diff
@@ -44,6 +44,7 @@ message NetworkEnvelope {
 
         OpenNewDisputeMessage open_new_dispute_message = 22;
         PeerOpenedDisputeMessage peer_opened_dispute_message = 23;
+        // ChatMessage in java class. We cannot rename protobuf definition because it would break backward compatibility
```

Comment:
does it really break compatibility? i think protobuf uses just the id.

Reply:
I just tested it. With pure network messages the class and field name are irrelevant, so yes in that case we can rename it. I will make a commit later.
Though we have to take care to not do that on protobuf objects which are peristed (or used in both network and peristence context) as locally stored data would break then.

Reply:
because we use the class name in our local storage code?

Reply:
Ups, was wrong thought... Yes is same with peristed data.

Reply:
@christophsturm Enums must not be renamed as we use the name for resolving.